floodviz/hydrograph_utils.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+import time
+
+
+def parse_hydrodata(jdata):
+    
+    """ 
+    Parses json Hydrodata from NWIS webservice
+    and formats for NVD3 charting library.
+
+    ARGS: 
+        jdata - json containing time series data for all sites listed in config.py SITE_IDs
```

Comment:
Should be specific about what types you are expecting. For instance this is a list of dict